sdk/ai/azure-ai-projects/azure/ai/projects/operations/_patch.py

```diff
@@ -183,7 +187,7 @@ def get_embeddings_client(self, **kwargs) -> "EmbeddingsClient":
         return client
 
     @distributed_trace
-    def get_azure_openai_client(self, *, api_version: str | None = None, **kwargs) -> "AzureOpenAI":
+    def get_azure_openai_client(self, *, api_version: Optional[str] = None, **kwargs) -> "AzureOpenAI":
         """Get an authenticated AzureOpenAI client (from the `openai` package) for the default
```

Comment:
You will have merge conflicts here, as I just checked in tis fix: https://github.com/Azure/azure-sdk-for-python/pull/38344 . Not sure which is the right change to keep, yours or mind. Is "Optional[]" support by Python 3.9/3.8?

Reply:
Yes, Union[<type>, None] is sinonimous to Optional[<type>]